src/vm/i386/excepx86.cpp

```diff
@@ -38,6 +38,7 @@
 #include "asmconstants.h"
 #include "virtualcallstub.h"
 
+#ifndef WIN64EXCEPTIONS
```

Comment:
Is it better to another implementation of ``CLRNoCatchHandler``? 

``EX_END_NOCATCH`` in ``vm/clrex.h`` uses ``CLRNoCatchHandler`` and we need to in order to resolve undefined reference issues.

It would be the best if we could eliminate the use of ``CLRNoCatchHandler``, but I'm not sure whether it is possible.